net-irc/inspircd/inspircd-2.0.26.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit toolchain-funcs user eapi7-ver
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
+HOMEPAGE="https://inspircd.github.com/"
+SRC_URI="https://github.com/inspircd/inspircd/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="geoip gnutls ipv6 ldap mysql pcre posix postgres sqlite ssl tre"
+
+RDEPEND="
+	dev-lang/perl
+	ssl? ( dev-libs/openssl:= )
+	geoip? ( dev-libs/geoip )
+	gnutls? ( net-libs/gnutls:= dev-libs/libgcrypt:0 )
+	ldap? ( net-nds/openldap )
+	mysql? ( virtual/mysql )
+	postgres? ( dev-db/postgresql:= )
+	pcre? ( dev-libs/libpcre )
+	sqlite? ( >=dev-db/sqlite-3.0 )
+	tre? ( dev-libs/tre )"
+DEPEND="${RDEPEND}"
+
+DOCS=( docs/. )
+PATCHES=( "${FILESDIR}"/${P}-fix-path-builds.patch )
+
+pkg_setup() {
+	enewgroup ${PN}
+	enewuser ${PN} -1 -1 -1 ${PN}
+}
+
+src_prepare() {
+	default
+
+	# Patch the inspircd launcher with the inspircd user
+	sed -i -e "s/@UID@/${PN}/" "${S}/make/template/${PN}" || die
+}
+
+src_configure() {
+	local extras=""
+
+	use geoip && extras+="m_geoip.cpp,"
+	use gnutls && extras+="m_ssl_gnutls.cpp,"
+	use ldap && extras+="m_ldapauth.cpp,m_ldapoper.cpp,"
+	use mysql && extras+="m_mysql.cpp,"
+	use pcre && extras+="m_regex_pcre.cpp,"
+	use posix && extras+="m_regex_posix.cpp,"
+	use postgres && extras+="m_pgsql.cpp,"
+	use sqlite && extras+="m_sqlite3.cpp,"
+	use ssl && extras+="m_ssl_openssl.cpp,"
+	use tre && extras+="m_regex_tre.cpp,"
+
+	# The first configuration run enables certain "extra" InspIRCd
+	# modules, the second run generates the actual makefile.
+	if [[ -n "${extras}" ]]; then
+		econf --disable-interactive --enable-extras=${extras%,}
```

Comment:
econf is not correct here:
```
Unknown option: build
Unknown option: host
Unknown option: mandir
Unknown option: infodir
Unknown option: datadir
Unknown option: sysconfdir
Unknown option: localstatedir
Unknown option: libdir
```

econf is meant for autotools and things that are really compatible with autotools. Call `./configure` directly.

Reply:
Okay, done.